### PR TITLE
docs: Update website with migration note

### PR DESCRIPTION
**Pull request by burakukula**
Adding the note on the website about the color update.
Pls, help me out with the copy 🙂

Screenshot 2021-07-22 at 10 34 01

+ changelog update
+ fix of the version in the migration notes

### CODE REVIEW

File: packages/forma-36-website/gatsby-config.js [hunk 1]
```diff
@@ -3,10 +3,11 @@ const path = require('path');
 module.exports = {
   siteMetadata: {
     title: 'Forma 36 - The Contentful Design System',
-    promoText: '',
-    promoLink: '',
-    promoLinkText: '',
-    promoTagText: '',
+    promoText: 'Updated color system.',
```
Comment (denkristoffer) [inline packages/forma-36-website/gatsby-config.js:6]:
```suggestion
    promoText: `Updated color system with new tokens.`,
```

File: packages/forma-36-tokens/CHANGELOG.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 
-* new colors tokens update ([#1041](https://github.com/contentful/forma-36/issues/1041)) ([fc0beba](https://github.com/contentful/forma-36/commit/fc0beba32310d2ab31e59006f6701f6ab7c0f79f))
+* Updated color system that contains new set of tokens with new namespaces and codemod to improve the experience of the migration ([#1041](https://github.com/contentful/forma-36/issues/1041)) ([fc0beba](https://github.com/contentful/forma-36/commit/fc0beba32310d2ab31e59006f6701f6ab7c0f79f))
```
Comment (denkristoffer) [inline packages/forma-36-tokens/CHANGELOG.md:11]:
```suggestion
* Updated color system that contains a new set of tokens with new namespaces ([#1041](https://github.com/contentful/forma-36/issues/1041)) ([fc0beba](https://github.com/contentful/forma-36/commit/fc0beba32310d2ab31e59006f6701f6ab7c0f79f))
```

I don't think the codemod is a part of the tokens so I'd say we should mention it separately but still keep it easy to find.

**Review by denkristoffer — CHANGES_REQUESTED** [review]

File: packages/forma-36-tokens/CHANGELOG.md (hunk 2 above)
Comment (burakukula) [reply packages/forma-36-tokens/CHANGELOG.md:11]: yeah, it was part of the same PR, thats' why I thought I would add it here. Do you suggest a separate mention of in below? How it should look like in the changelog?

Comment (burakukula) [reply packages/forma-36-tokens/CHANGELOG.md]: yeah, sorry, codemod has its own changelog.

**Review by denkristoffer — APPROVED** [review]

**Review by massao — APPROVED** [review]

**Pull request merged**